client.go

```diff
@@ -169,8 +169,11 @@ func (c *client) authorize(ctx context.Context, principal string, check AthenzAc
 	if err != nil {
 		authzResponse.Granted = false
 		if err, ok := err.(*statusCodeError); ok {
-			if err.code == http.StatusNotFound {
+			switch err.code {
+			case http.StatusBadRequest:
```

Comment:
Can we add an additional check to authz.go here: https://github.com/yahoo/k8s-athenz-webhook/blob/master/authz.go#L227-L230 to check for http.StatusBadRequest and directly deny the user right away with a clear error message?

Reply:
Sure. Added. Please review again. Thanks.